<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Dialog</class>

	
  <widget class="QDialog" name="Dialog">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>700</height>
      </rect>
    </property>

    <property name="windowTitle">
      <string>Farmin Building variation detection</string>
    </property>

    <widget class="QPushButton" name="btnLoadImage">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>10</y>
          <width>711</width>
          <height>25</height>
        </rect>
      </property>
      <property name="text">
        <string>Load Image</string>
      </property>
    </widget>

    <widget class="QLabel" name="lblImage">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>45</y>
          <width>351</width>
          <height>250</height>
        </rect>
      </property>
      <property name="frameShape">
        <enum>QFrame::Box</enum>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>

    <widget class="QLabel" name="lblImage2">
      <property name="geometry">
        <rect>
          <x>380</x>
          <y>45</y>
          <width>351</width>
          <height>250</height>
        </rect>
      </property>
      <property name="frameShape">
        <enum>QFrame::Box</enum>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>

    <widget class="QLabel" name="lblOutputText">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>295</y>
          <width>351</width>
          <height>22</height>
        </rect>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>

    <widget class="QSlider" name="cannySlider">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>312</y>
          <width>711</width>
          <height>22</height>
        </rect>
      </property>
      <property name="orientation">
        <enum>Qt::Horizontal</enum>
      </property>
      <property name="minimum">
        <number>100</number>
      </property>
      <property name="maximum">
        <number>255</number>
      </property>
      <property name="tickInterval">
        <number>10</number>
      </property>
    </widget>


    <widget class="QPushButton" name="btnApplyCanny">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>330</y>
          <width>711</width>
          <height>25</height>
        </rect>
      </property>
      <property name="text">
        <string>Detect Changes</string>
      </property>
    </widget>

    <widget class="QLabel" name="lblCannyImage">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>360</y>
          <width>351</width>
          <height>300</height>
        </rect>
      </property>
      <property name="frameShape">
        <enum>QFrame::Box</enum>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>

    <widget class="QLabel" name="lblNoOfContours">
      <property name="geometry">
        <rect>
          <x>380</x>
          <y>468</y>
          <width>351</width>
          <height>24</height>
        </rect>
      </property>
      <property name="frameShape">
        <enum>QFrame::Box</enum>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>

    <widget class="QLabel" name="lblFinalThreshold">
      <property name="geometry">
        <rect>
          <x>380</x>
          <y>500</y>
          <width>351</width>
          <height>24</height>
        </rect>
      </property>
      <property name="frameShape">
        <enum>QFrame::Box</enum>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>



	<widget class="QLabel" name="lblCars">
      <property name="geometry">
        <rect>
          <x>380</x>
          <y>530</y>
          <width>351</width>
          <height>24</height>
        </rect>
      </property>
      <property name="frameShape">
        <enum>QFrame::Box</enum>
      </property>
      <property name="text">
        <string />
      </property>
    </widget>
	
	
	
    <widget class="QPushButton" name="btnSaveImage">
      <property name="geometry">
        <rect>
          <x>20</x>
          <y>665</y>
          <width>711</width>
          <height>25</height>
        </rect>
      </property>
      <property name="text">
        <string>Save Image</string>
      </property>
    </widget>
  </widget>
  <resources />
  <connections />
</ui>